group_vars/maintenance.yml

```
@@ -0,0 +1,513 @@
+#       - data_format = "influx"
+#       - interval = "1m"
+#   galaxy_oidc:
+#     plugin: "exec"
+#     config:
+#       - commands = ["{{ custom_telegraf_env }} /usr/bin/gxadmin iquery users-with-oidc"]
+#       - timeout = "15s"
+#       - data_format = "influx"
+#       - interval = "1m"
+#   galaxy_workflow:
+#     plugin: "exec"
+#     config:
+#       - commands = ["{{ custom_telegraf_env }} /usr/bin/gxadmin iquery workflow-invocation-status"]
+#       - timeout = "15s"
+#       - data_format = "influx"
+#       - interval = "1m"
+#   galaxy_workflow_totals:
+#     plugin: "exec"
+#     config:
+#       - commands = ["{{ custom_telegraf_env }} /usr/bin/gxadmin iquery workflow-invocation-totals"]
+#       - timeout = "15s"
+#       - data_format = "influx"
+#       - interval = "1m"
+#   # Uses logs from /var/log/nginx/ so it should be run only on the headnode
+#   # galaxy_active_users:
+#   #   plugin: "exec"
+#   #   config:
+#   #     - commands = ["/usr/bin/gxadmin local cu"]
+#   #     - timeout = "15s"
+#   #     - data_format = "influx"
+#   #     - interval = "1m"
+#   postgres_extra:
+#     plugin: "exec"
+#     config:
+#       - commands = [
+#         "{{ custom_telegraf_env }} /usr/bin/gxadmin iquery pg-cache-hit",
+#         "{{ custom_telegraf_env }} /usr/bin/gxadmin iquery pg-index-size",
+#         "{{ custom_telegraf_env }} /usr/bin/gxadmin iquery pg-index-usage",
+#         "{{ custom_telegraf_env }} /usr/bin/gxadmin iquery pg-table-bloat",
+#         "{{ custom_telegraf_env }} /usr/bin/gxadmin iquery pg-table-size",
+#         "{{ custom_telegraf_env }} /usr/bin/gxadmin iquery pg-unused-indexes",
+#         "{{ custom_telegraf_env }} /usr/bin/gxadmin iquery pg-vacuum-stats",
+#         "{{ custom_telegraf_env }} /usr/bin/gxadmin iquery pg-stat-bgwriter",
+#         "{{ custom_telegraf_env }} /usr/bin/gxadmin iquery pg-stat-user-tables",
+#         ]
+#       - timeout = "60s"
+#       - data_format = "influx"
+#       - interval = "2m"
+
+# Custom pip installer
+pip_venv_path: "{{ galaxy_venv_dir }}"
+pip_install_dependencies:
+  # phdcomics
+  - beautifulsoup4
+  # Used by code_files for apollo
+  - biopython
+  - bcbio-gff
+  # Needed for tool watching
+  - watchdog
+  # Needed gxadmin graphs
+  - data_hacks
+  # used by reports
+  - WeasyPrint
+  - nbconvert
+  - gitlab-arc-fs
+  # Needed for usegalaxy-eu.vgcn-monitoring Telegraf role
+  - pyyaml
+  - GitPython
+  - python-openstackclient
+
+yum_exclude_repos:
+  - condor*
+  - npm*
+
+all_yum_repositories:
+  - name: galaxy_gpel
+    description: Galaxy Packages for Enterprise Linux $releasever - $basearch
+    baseurl: https://depot.galaxyproject.org/yum/el/$releasever/$basearch/
+    enabled: true
+    gpgcheck: false
+    retries: 1
+    timeout: 10
+
+# Galaxy monitoring
+monitor_condor: true
+monitor_condor_split_util: true
+
+# Rsync to NFS and to the headnodes
+# Role: usegalaxy-eu.rsync-galaxy-sync
+galaxy_nfs_location: /data/galaxy-sync
+
```

```suggestion
execute_galaxy_sync: false

```
makes it a bit more clear and can be changed to true later

---

It's default, however I have added it to the group_vars with a comment describing it for more verbosity.